api/applicationStages.ts

```diff
@@ -0,0 +1,40 @@
+import { ApiPromise, EmptyPayload } from '../types/apiResponse';
```

Comment:
Can we rename the file to be *Api.ts to be consistent with the other files in this folder?